sdk/core/scheduler/main.cc

```diff
@@ -668,29 +650,18 @@ __cheriot_minimum_stack(0xc0) int multiwaiter_wait(Timeout           *timeout,
 namespace
 {
 	/**
-	 *
+	 * An interrupt capability.
 	 */
-	struct InterruptCapability : Handle
+	struct InterruptCapability : Handle</*IsDynamic=*/false>
 	{
-		/**
-		 * Type marker used by `Handle`, tells it to use the dynamic path.
-		 */
-		static constexpr auto TypeMarker = Handle::Type::Dynamic;
-
 		/**
 		 * Dynamic type marker used by `Handle`.
 		 */
-		static Capability<void> dynamic_type_marker()
+		static SKey dynamic_type_marker()
```

Comment:
I understand (now) that this is meant as "(dynamic type) marker", but maybe just call it `sealing_type` or `sealing_key` by similarity with `STATIC_SEALING_TYPE` or `SKey`/`InterruptKey`/`MultiWaiterKey`.

Reply:
Probably a good idea.  The 'dynamic' name is left over from the internal version where we had both static and dynamic types.